File: Backend/TrackrAPI/TrackrAPI/Dtos/Chats/ChatDTO.cs
namespace TrackrAPI.Models;
public partial class ChatDTO
{

    public int IdChat { get; set; }
    public DateTime Fecha { get; set; }
    public bool Habilitado { get; set; }
    public string? Titulo { get; set; }
    public string ImagenBase64 { get; set; }
    public string TipoMime { get; set; }

}

File: Frontend/trackr-paciente/src/app/shared/Dtos/Chat/chat-dto.ts
export interface ChatDTO{
    idChat?: number;
    fecha: string;
    fechaYaFormateada?: boolean;
    habilitado: boolean;
    titulo?: string;
    ultimoMensaje?:string
    fechaUltimoMensaje?: string;
    idCreadorChat: number;
    imagenBase64?: string;
    tipoMime?: string;
    urlImagen? : any;
}